test/run/test_mount.py

```diff
@@ -65,7 +65,11 @@ def create_image(tmpdir):
         "options": {
             "label": "TEST",
             "volid": "7B7795E7",
-            "fat-size": 32
+            "fat-size": 32,
+            "geometry": {
```

Comment:
I think there should be a separate test for this behavior, it's also going to need to check the geometry to make sure it showed up as expected, not just that it created it without error.

Reply:
Thanks for your review. I just pushed a new version with a unit test for this. Let me know if that's what you had in mind :)

Reply:
I like the new unit test, thanks. But I don't think this needs to be in all the calls to create_image. Maybe add a flag to test with it and without it? Assume the normal case will be that it isn't included, so that needs to continue to be tested. @thozza  what do you think?

Reply:
I agree that we also want to keep testing cases when the geometry is not specified. So making this optional and using it in a dedicated test case is preferred from my PoV.

Reply:
Thanks for your feedback. I just pushed a new version removing the geometry parameters in `test/run/test_mount.py`